model/patch/patch.go

```diff
@@ -144,7 +144,7 @@ type Patch struct {
 	VariantsTasks   []VariantTasks         `bson:"variants_tasks"`
 	SyncAtEndOpts   SyncAtEndOptions       `bson:"sync_at_end_opts,omitempty"`
 	Patches         []ModulePatch          `bson:"patches"`
-	Parameters      []Parameter            `bson:"parameters"`
+	Parameters      []Parameter            `bson:"parameters,omitempty"`
```

Comment:
Have you checked that there aren't any queries doing $size that this change would break?

Reply:
Yep, I also looked at instances where Parameters is used and I can't find anything that would break with nil instead of the empty list so I think it's good.